packages/suite-desktop/src/modules/bridge.ts

```diff
@@ -20,6 +34,37 @@ const load = async () => {
         bridge.stop();
     });
 
+    ipcMain.handle('bridge/toggle', async (_: unknown) => {
+        const status = await handleBridgeStatus(bridge, mainWindow);
+        try {
+            if (status.service) {
+                await bridge.stop();
+                store.setBridgeSettings({ startOnStartup: false });
+            } else {
+                await bridge.start();
+                store.setBridgeSettings({ startOnStartup: true });
+            }
+            return { success: true };
+        } catch (error) {
+            return { success: false, error };
+        } finally {
+            handleBridgeStatus(bridge, mainWindow);
+        }
+    });
+
+    ipcMain.handle('bridge/get-status', async () => {
+        try {
+            const status = await bridge.status();
+            return { success: true, status };
```

Comment:
If type would be corrected, this'll be `{ success: true, payload: status }` instead.

Also, in equivalent code in tor module we don't return a value from `tor/get-status` but emit `tor/status` instead, which allows us to handle the status in exactly the same manner whether it was demanded or not. I'm not saying it's necessarily better, but let's discuss it.

Reply:
fixed in 2b92f68103839b87ef3e5996f8d1a686b2235eda

I somewhat dislike that `something/get-something` triggers firing of event instead of returnig something directly :thinking: hard to follow. But I am open to discuss it in the office